packages/backend/src/handlers/actions/guild/saveGuildHandler.ts

```diff
@@ -0,0 +1,86 @@
+import { GuildDiscordMetadata } from '@metafam/discord-bot';
+import { Request, Response } from 'express';
+
+import {
+  Guild_Set_Input,
+  GuildInfo,
+  GuildType_Enum,
+  SaveGuildResponse,
+} from '../../../lib/autogen/hasura-sdk';
+import { client } from '../../../lib/hasuraClient';
```

Comment:
Think we should add absolute path support in a future PR. This is becoming ridiculous lol